src/NServiceBus.Core/Persistence/Learning/LearningSynchronizedStorageSession.cs

```diff
@@ -85,11 +85,5 @@ public Task Complete(IContainSagaData sagaData)
         Dictionary<string, SagaStorageFile> sagaFiles = new Dictionary<string, SagaStorageFile>();
 
         List<StorageAction> deferredActions = new List<StorageAction>();
-
-        static class DefaultSagaData<TSagaData>
```

Comment:
Same comment here as the for the version of this class in the in-memory persister.

Reply:
Don't think we need this since we return `null` instead of `default` so there is no task to cache here?

Reply:
Yeah, looking at this one again, this one is OK to remove since it's being used in an async method, so we're returning null directly.